_appmap/test/test_env.py

```diff
@@ -11,3 +12,17 @@ def test_disable_temporarily():
     except RuntimeError:
         ...
     assert env.enables("requests")
+
+def test_determine_log_file(monkeypatch):
+    env = Env({"_APPMAP": "true"})
+
```

Comment:
I think it's important to add an end-to-end test that actually shows that the behavior described in #339 is fixed. It would be ideal to add this to the smoketest, but that's going to be complicated by the fact that processes in docker run as root.

Can you give some thought to how we might implement such a test, please?